merqube_client_lib/pydantic_v2_types.py

```diff
@@ -663,13 +663,13 @@ class Root(Enum):
 
 
 class ClientBufferSpecific(BaseModel):
-    buffer_start: confloat(ge=0.0, le=1.0) = Field(
+    protection_start: confloat(ge=0.0, le=100.0) = Field(
         ...,
-        description="losses on the underlying are incurred up to (1-buffer_start), the portfolio is protected between (1-start) and (1-end), and then losses after (1-end) are also incurred. Eg on start=0.95 and end=0.7 (a so called 5/30 buffer), the first 5% of losses are incurred, the portfolio is protected between 5 and 30, then losses after 30 are added to 5 and are incurred.",
-        example=0.95,
+        description="losses on the underlying are incurred up to protection_start, the portfolio is protected against losses between protection_start and protection_end, and then losses after protection_end are also incurred. Eg on start=95 and end=70 (sometimes a called a 5/30 buffer), the first 5% of losses are incurred, the portfolio is protected between -5% and -30%, then losses after 30% are added to 5% and are incurred. In this example, if the underlying drops 32%, the index will lose 7%.",
```

Comment:
Validated

Reply:
does this description make sense? im just a janitor here